Sources/CorePayments/Networking/GraphQL/GraphQLClient.swift

```diff
@@ -1,17 +1,17 @@
 import Foundation
 
-class GraphQLClient {
+public class GraphQLClient {
 
-    private let environment: Environment
-    private let urlSession: URLSessionProtocol
-    private let jsonDecoder = JSONDecoder()
+    public let environment: Environment
+    public let urlSession: URLSessionProtocol
+    public let jsonDecoder = JSONDecoder()
 
     public init(environment: Environment, urlSession: URLSessionProtocol = URLSession.shared) {
         self.environment = environment
         self.urlSession = urlSession
     }
 
-    func executeQuery<T: Decodable>(query: GraphQLQuery) async throws -> GraphQLQueryResponse<T> {
+    func executeQuery<T: Decodable, Q: GraphQLQuery>(query: Q) async throws -> GraphQLQueryResponse<T> {
```

Comment:
It looks like previously `executeQuery` was used to perform the actual GraphQL request ([here](https://github.com/paypal/paypal-ios/blob/257df315bb8ab2e323df39bf17a5af48e36498a1/Sources/CorePayments/EligibilityAPI.swift#L37)). Now we are basically replacing it with `callGraphQL`. We should not need both of these functions.

Reply:
I just added optional name. I will delete it.

Reply:
I mean, now there are 2 functions that basically do the same exact thing.

Reply:
Addressed in this commit 6d3f15c04e1b44ab49f8dcc1575456180d2dbc69